lib/cocoapods/command/repo/push.rb

```diff
@@ -45,7 +45,7 @@ def initialize(argv)
           @local_only = argv.flag?('local-only')
           @repo = argv.shift_argument
           @source = source_for_repo
-          @source_urls = argv.option('sources', config.sources_manager.all.map(&:url).join(',')).split(',')
+          @source_urls = argv.option('sources', config.sources_manager.all.map(&:url).append(Pod::TrunkSource::TRUNK_REPO_URL).uniq.join(',')).split(',')
```

Comment:
this will add `trunk` even when a list of sources was specified, which is probably not what we want.

when doing `--sources=source1,source2` then we shouldn't be appending `trunk` to that list.

how about something like this:

```ruby
if @source_urls.empty?
  # The user hasn't added trunk as a source yet
  @source_urls = [Pod::TrunkSource::TRUNK_REPO_URL]
end
```

Reply:
actually I misread this, that was part of the default arg so this should be fine